app/forms/hyrax/default_work_form.rb

```diff
@@ -4,5 +4,29 @@ module Hyrax
   class DefaultWorkForm < Hyrax::Forms::WorkForm
     self.model_class = ::DefaultWork
     self.terms += [:resource_type]
+    self.terms -= [:date_created]
+
+    def self.build_permitted_params
+      permitted = super
+      date_terms.each do |date_term|
+        permitted << {
+          date_term => []
+        }
+      end
+    end
+
+    def date_terms
+        [
+          :date_created,
```

Comment:
Is `date_created` supposed to be a single value field? 

@vantuyls might help to verify the multi/single valuedness of each of these fields.